src/object.d

```diff
@@ -3241,6 +3241,7 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+// Compare floating point numbers for ordering (one instantiation per width).
```

Comment:
This is a tricky one: the `const` in the parameters actually does not change `F` - only the type of `f1` / `f2`.